js/common/modules/@arangodb/foxx/manager-utils.js

```diff
@@ -219,7 +219,7 @@ function getServiceDefinition (mount) {
 }
 
 // //////////////////////////////////////////////////////////////////////////////
-// / @brief Update the app installed at this mountpoint with the new app
+// / @brief Update the app installed at this mount point with the new app
```

Comment:
do we have mountpoint variables etc. that really should be mountPoint?

Reply:
It is actually called `mountPoint` **sometimes**:
- `js\apps\system\_admin\aardvark\APP\frontend\js\arango\arango.js`
- some frontend views
- `js\server\modules\@arangodb\actions.js`
- `js\server\modules\@arangodb\foxx\manager.js`

There are quite a few constants that contain `MOUNTPOINT`. Not sure if we want to change that to `MOUNT_POINT` for consistency...